third_party/xla/xla/service/gpu/conv_algorithm_picker.cc

```diff
@@ -71,7 +71,7 @@ limitations under the License.
 #include "tsl/platform/logging.h"
 #include "tsl/platform/numbers.h"
 #include "tsl/platform/statusor.h"
-#include "tsl/util/env_var.h"
+#include "xla/tsl/util/env_var.h"
```

Comment:
I think this `env_var.h` is only used in our side due to https://github.com/ROCm/tensorflow-upstream/pull/2467 and it's merged https://github.com/ROCm/tensorflow-upstream/commit/c3d966e559ffd13f6a473d1b57e606314a456ce8 maybe you can upstream that PR as well? cc @pemeliya